teuthology/misc.py

```diff
@@ -154,6 +154,16 @@ def get_testdir(ctx=None):
     )
 
 
+def get_tmpdir():
+    """
+    :returns: A test directory
+    """
+    return config.get(
+        'tmp_path',
+        '/tmp/cephtest'
+    )
+
+
```

Comment:
Why this method exist at all?

Reply:
If the purpose to have change default temporary directory for remotes, this method should go to orchestra remote and return by default None, and the `/tmp/cephtest` value should be set in teuthology config then.

Reply:
It exists here because that seemed to be the most coherent place for it. e.g. `get_testdir`, `get_archive_dir`, etc.

Reply:
it is not the most, as I said, this is not a temp path for teuthology itself, this is used for remote nodes only. But in general I don't see any advantage of having this proxy method at all.

Reply:
Uh, these `get_` methods are all for paths on the test nodes themselves. I don't see how it's "for teuthology itself".

Reply:
Also in other comments, the `get_tmpdir` does not say to user anything, is this local temp dir where teuthology is running? Or is it only related to some specific test? Why there is a hardcoded name like `/tmp/cephtest`? Why not `/tmp/teuthology`? Teuthology as a test framework and common modules should be designed and developed as not hardly bounded to ceph, it is hard to believe, but it could run not only ceph specific stuff.

Reply:
> Uh, these `get_` methods are all for paths on the test nodes themselves. I don't see how it's "for teuthology itself".

I'm not following.